### PR TITLE
[10.0][IMP] Add display sequence on contract lines

### PR DESCRIPTION
**Pull request by pedrocs-exo**
A simple improvement to allow users to reorder the contract lines.

### CODE REVIEW

**Review by carlosdauden — CHANGES_REQUESTED** [review]
Change version number

**Comment by pedrocs-exo** [comment]
Current version is 10.0.3.0.0. Should I bump it to 10.0.3.0.1 or 10.0.3.1.0?

**Comment by carlosdauden** [comment]
At least 10.0.3.1.0

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Comment by pedrocs-exo** [comment]
Thank you for the info. The PR has been updated.

**Review by pedrobaeza — APPROVED** [review]
Squashing on merge

File: contract/models/account_analytic_invoice_line.py [hunk 1]
```diff
@@ -54,6 +55,11 @@ class AccountAnalyticInvoiceLine(models.Model):
         help='Discount that is applied in generated invoices.'
              ' It should be less or equal to 100',
     )
+    sequence = fields.Integer(
+        string="Sequence",
+        default=10,
+        help="Sequence of the contract line when displaying contracts"
```
Comment (lasley) [inline contract/models/account_analytic_invoice_line.py]: Please add trailing comma

**Review by lasley — APPROVED** [review]
Great improvement, thanks @pedrocasi. One comment inline, minor

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**